src/In.ProjectEKA.HipService/Discovery/PatientDiscovery.cs

```diff
@@ -32,24 +32,25 @@ public class PatientDiscovery : IDiscovery
 
         public async Task<Tuple<DiscoveryRepresentation, ErrorRepresentation>> PatientFor(DiscoveryRequest request)
         {
-            var (linkRequests, exception) = await linkPatientRepository.GetLinkedCareContexts(request.Patient.Id);
+            var (linkedAccounts, exception) = await linkPatientRepository.GetLinkedCareContexts(request.Patient.Id);
             if (exception != null)
             {
                 return new Tuple<DiscoveryRepresentation, ErrorRepresentation>(null,
                     new ErrorRepresentation(new Error(ErrorCode.FailedToGetLinkedCareContexts,
                         "Failed to get Linked Care Contexts")));
             }
 
-            if (HasAny(linkRequests))
+            var linkedCareContexts = linkedAccounts.ToList();

```

Comment:
Does it need to be there?

Reply:
Except migration could have been handled little better. Can you check how we can keep the data or will this code change handle it properly? I doubt it.

Reply:
var linkedCareContexts = linkedAccounts.ToList(); ---> This was done as it was throwing "Possible multiple enumeration" Warning

Reply:
Except migration could have been handled little better. Can you check how we can keep the data or will this code change handle it properly? I doubt it.

------------> I am not able to understand this?

Reply:
If you have data in the table, what would happen after the changes?

Reply:
which data ?  Linked Accounts ? then nothing will happen